tools/replaceColumn/replaceColumn.xml

```diff
@@ -63,14 +74,27 @@ with open("output_file", 'w') as output:
                label="Which column should be replaced?" />
         <param name="skip_lines" type='integer' value='0' label="Skip this many starting lines" />
         <param name="delimiter" type="select" label="Delimited by">
-            <option value="" selected="True">Tab</option>
-            <option value=" ">Whitespace</option>
+            <option value="" selected="True">Whitespace</option>
+            <option value="  ">Tab</option>
```

Comment:
I agree it should be the default, but Whitespace was the default before (even though it was labeled "Tab").  I'll change it, but I don't know if it will inadvertently break any workflows that assume whitespace behavior.